cfgov/research_reports/models.py

```diff
@@ -40,10 +40,19 @@ def sec(request, page):
             'url': format('#section-{}', i),
             'numbering': format('{}: ' if is_appendix else '{}. ', i),
             'children': [{
-                'title': subsection.sub_header,
-                'body': subsection.sub_body,
+                'title': subsection.header,
+                'body': subsection.body,
                 'url': format('#section-{}.{}', i, j),
-                'numbering': '' if is_appendix else format('{}.{} ', i, j)
+                'numbering': '' if is_appendix else format('{}.{} ', i, j),
+                'children': [{
+                    'title': section3.header,
+                    'body': section3.body,
+                    'url': format('#section-{}.{}.{}', i, j, k),
+                    'numbering': '' if is_appendix else
+                                 format('{}.{}.{} ', i, j, k),
+                } for k, section3 in enumerate(
+                    subsection.report_sections_level_three.all().order_by('pk')
```

Comment:
The whole `get_report_parts` method is getting a little gnarly. I'll think about how to make it cleaner.